pkg/tracker/currentVariables.go

```diff
@@ -78,7 +78,7 @@ func (b *CurrentVariablesTracker) Exec(ctx context.Context) error {
 		}
 	}
 
-	err = b.db.Put(db.DifficultyKey, []byte(hexutil.EncodeBig(returnNewVariables.Difficutly)))
+	err = b.db.Put(db.DifficultyKey, []byte(hexutil.EncodeBig(returnNewVariables.Difficulty)))
```

Comment:
@JGcarv I imagine this was changed with the latest contact upgrade?

Reply:
Yes, that should be fixed!